### PR TITLE
implement jobs page, split subscribe section and footer, add headerMode

### PR DESCRIPTION
**Pull request by mailtime-wing**
1. Implement jobs page.
2. Split old subscribe section that contains social icons and other pages into footer.
3. Added header mode in Layout component(default is dark mode). 
e.g. `<Layout headerMode="light">`

please review, thanks!

### CODE REVIEW

File: src/pages/jobs.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      ))}
+    </div>
+  )
+}
+
+const JobPage = () => {
+  const theme = useTheme()
+  return (
+    <Layout headerMode="light">
+      <SEO title="Press" />
```
Comment (chochihim) [inline src/pages/jobs.js]: should not be "Press"?

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -80,62 +103,138 @@ const reducer = (state, action) => {
...
+                          defaultMessage="Leave a message"
+                        />
+                      }
+                    />
+                  </>
+                )}
+              </div>
+            </>
+          </Popup>
+        </div>
```
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/components/Footer/index.js]: done.

File: src/components/JobSection/Hiring/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from "react"
```
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/components/JobSection/Hiring/index.js]: moved.

File: src/components/JobSection/Job/index.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React from "react"
```
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/components/JobSection/Job/index.js]: moved.

File: src/components/JobSection/Hiring/index.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+}) => (
+  <div {...props}>
+    <h1 css={title}>{header}</h1>
+
+    <p css={theme => [contentFontStyle(theme), content]}>{description}</p>
+    <p css={theme => [contentFontStyle(theme), emailMargin]}>{email}</p>
+    <p css={contentFontStyle}>{currentOpenings}</p>
+    <div css={currentOpeningsContainer}>
+      {openingJobs.map((openingJob, index) => (
+        <>
```
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/components/JobSection/Hiring/index.js]: already added in `<span css={jobTitle} key={openingJob.toString()}>`.

File: src/components/JobSection/Hiring/index.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  header,
+  description,
+  email,
+  currentOpenings,
+  openingJobs,
+  ...props
+}) => (
+  <div {...props}>
+    <h1 css={title}>{header}</h1>
+
```
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/components/JobSection/Hiring/index.js]: removed.

File: src/components/JobSection/jobs.json [hunk 7]
```diff
@@ -0,0 +1,188 @@
+[
```
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/components/JobSection/jobs.json]: moved.

File: src/components/SlopedBackground/index.js [hunk 8]
```diff
@@ -73,10 +73,10 @@ const Navigator = ({ mode }) => {
   )
 }
 
-const SlopedSection = ({ children, mode, ...props }) => (
+const SlopedSection = ({ children, mode, isOtherPage, ...props }) => (
   <Background {...props}>
     <Navigator mode={mode} />
-    <div css={content}>{children}</div>
+    <div css={content(isOtherPage)}>{children}</div>
```
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/components/SlopedBackground/index.js]: done.

File: src/pages/jobs.js (hunk 1 above)
Comment (mailtime-wing) [reply src/pages/jobs.js]: typo and changed.

File: src/components/JobSection/Hiring/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (chochihim) [reply src/components/JobSection/Hiring/index.js]: You should add to `<>`. Please read the warning from console

File: src/components/Footer/index.js [hunk 9]
```diff
@@ -117,127 +117,129 @@ const Page = ({ children, ...props }) => (
   </Link>
 )
 
-const Footer = () => {
+const UpperFooter = () => (
```
Comment (chochihim) [inline src/components/Footer/index.js]: `<UpperFooter />` and `<LowerFooter />` are quite large. Please put them into new files.

Comment (mailtime-wing) [reply src/components/Footer/index.js]: done.

File: src/components/JobSection/Hiring/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/components/JobSection/Hiring/index.js]: fixed.

File: src/pages/index.js [hunk 10]
```diff
@@ -22,7 +22,7 @@ const background = theme => css`
 `
 
 const lastSection = css`
```
[the remark this answers is not in the document]
Comment (mailtime-wing) [reply src/pages/index.js]: removed.

File: src/components/HiringSection/style.js [hunk 11]
```diff
@@ -41,3 +41,9 @@ export const jumpAnchor = css`
 export const jobContainer = css`
   display: inline-block;
 `
+
+export const emailStyle = theme => css`
```
Comment (chochihim) [inline src/components/HiringSection/style.js]: need `font-weight: bold;`

File: src/components/PressSection/style.js [hunk 12]
```diff
@@ -28,3 +28,16 @@ export const content2 = css`
 export const emailMargin = css`
   margin-bottom: 5rem;
 `
+
+export const emailStyle = theme => css`
```
Comment (chochihim) [inline src/components/PressSection/style.js]: need `font-weight: bold;`

File: src/pages/jobs.js [hunk 13]
```diff
@@ -36,104 +30,47 @@ const jobsSection = css`
...
+        <Hiring
+          css={hiringPadding}
+          openingJobs={rawData.map(
+            rawDataItem => rawDataItem.jobTitle[intl.locale]
+          )}
+        />
       </SlopedSection>
-      <JobsSection />
+      <div css={jobsSection}>
+        {useMemo(
```
Comment (chochihim) [inline src/pages/jobs.js]: do not inline `useMemo`

File: src/pages/jobs.js [hunk 14]
```diff
@@ -36,104 +30,47 @@ const jobsSection = css`
...
-            />
-          }
-          plus={<FormattedMessage id="jobPlus" defaultMessage="**Plus**" />}
-        />
-      ))}
-    </div>
-  )
-}
-
 const JobPage = () => {
```
Comment (chochihim) [inline src/pages/jobs.js]: Change this component to 
```js
const JobPage = () => {
  const theme = useTheme()
  const intl = useIntl()

  const data = useMemo(
    () =>
      rawData.map(rawDataItem => {
        const dataItem = {}
        Object.keys(rawDataItem).forEach(key => {
          if (rawDataItem[key].length !== undefined) {
            dataItem[key] = []
            Object.keys(rawDataItem[key]).forEach(sKey => {
              dataItem[key].push(rawDataItem[key][sKey][intl.locale])
            })
          } else {
            dataItem[key] = rawDataItem[key][intl.locale] || rawDataItem[key]
          }
        })
        return dataItem
      }),
    [intl.locale]
  )
  const openingJobs = useMemo(() => data.map(dataItem => dataItem.jobTitle), [data])

  return (
    <Layout headerMode="light">
      <SEO title="Jobs" />
      <SlopedSection
        css={background}
        slopedBackgroundImage={theme.linearGradients.greenDarkToLight}
      >
        <Hiring css={hiringPadding} openingJobs={openingJobs} />
      </SlopedSection>
      <div css={jobsSection}>
        {data.map((job, index) => (
          <Job key={index} jobDetails={job} />
        ))}
      </div>
    </Layout>
  )
}
```

and study what is changed.

**Comment by chochihim** [comment]
@mailtime-wing It's looking good now. Can you rebase to master?

**Pull request merged**